processfamily/__init__.py

```diff
@@ -102,8 +104,8 @@ def __init__(self, child_process):
         sys.stdout = open(os.devnull, 'w')
 
         self._stdout_lock = threading.RLock()
-        self._sys_in_thread = threading.Thread(target=self._sys_in_thread_target)
-        self._sys_in_thread.setDaemon(True)
+        self._sys_in_thread = threading.Thread(target=self._sys_in_thread_target, name="pf_%s_stdin" % repr(child_process))
+        self._sys_in_thread.daemon = True
```

Comment:
The `name` is just to make it easier to identify tracebacks and logs
The `daemon` API change was an update in `2.6`, and the docs refer to the `setDaemon` API as pre-2.6.